### DISCUSSION TITLE
Reproducing results with ancombc()

### DISCUSSION THREAD

**Discussion by ellimakar** [FrederickHuangLin/ANCOMBC#47] [Q&A] [answered]
Hello :)

I started exploring the ANCOM-BC and I am trying to reproduce the results from the article _Analysis of compositions of microbiomes with bias correction_ when comparing MA vs US at the 0-2 age group by using the ancombc() function. However, I get different results than those presented in the article...Not sure what I am missing but the code I am using is the following (after creating the phyloseq object):

test <- phyloseq(obs.abn_phyl, meta.data_phyl)

ancombc(phyloseq = test, formula = "country", 
                     p_adj_method = "bonferroni", zero_cut = 0.9, lib_cut = 1000,
                     group = "country", struc_zero = TRUE, neg_lb = TRUE, tol = 1e-5,
                     max_iter = 100, conserve = FALSE, alpha = 0.05, global = TRUE)

Thank you in advance!!
Elli

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi Elli,

Thanks for your interest in ANCOMBC!

The results from the paper you mentioned were produced using the standalone codes (ANCOMBC R package was not available at that time), which can be fully reproduced by this [repo](https://github.com/FrederickHuangLin/ANCOM-BC-Code-Archive).

We have updated the `ancombc` function since publication. Thus, it is expected to see some discrepancies, but the difference is supposed to be minimal.

Best,
Huang
